SU-SWS/jumpstart_ui
# Fixed the iframe aspect ratio styles to avoid conflicts

# READY FOR REVIEW

# Summary
- Fixed the aspect ratio selectors

# Need Review By (Date)
- asap

# Urgency
- critical

# Steps to Test
1. checkout this branch
1. create some cards/text areas etc with youtube videos.
1. ensure they aren't cut off and have a 16:9 ratio.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
